dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd

```diff
@@ -38,7 +38,7 @@ Import-Package: \
   *
 
 Export-Package: \
-	!com.ibm.ws.microprofile.health20.resources,\
+	com.ibm.ws.microprofile.health.resources,\
```

Comment:
Since the nls files are in the `com.ibm.ws.microprofie.health bundle`, why do we need to export the resources package here?

Reply:
I tried not exporting it, however, when the mpHealth-2.0 feature is being run, it throws a `Could not resolve module` error in the message.log. I think I need to export the resources package, so during runtime it gets picked up.

Reply:
Where do you specify the dependency of the common bundle?

Reply:
I added the common messages bundle under the `com.ibm.ws.microprofile.health` project. Since, I was already using some classes from the `com.ibm.ws.microprofile.health` package, I already had the following line `com.ibm.ws.microprofile.health;version=latest` in the build path of the bnd file for mpHealth-2.0.

Reply:
Since it isn't probably picked up automatically by bnd do you maybe need to add an explicit import for the package and that is why it was failing because it didn't know to have it in its import list?

Reply:
Oh I see.  We don't depend on com.ibm.ws.microprofile.health bundle in MP 2.x features. so yes it will need to be included in this bundle as well.  I wonder if you can put it as a private package in both the 2.0 and non 2.0 feature and it be included in the bundle and not have to be exported.

Reply:
Thanks Jared! Adding it as a private-package worked! Made a PR to fix it: https://github.com/OpenLiberty/open-liberty/pull/17533